### ISSUE TITLE
Failure on LispWorks 6 for Windows

### ISSUE THREAD

**Issue by quicklisp** [quicklisp/quicklisp-bootstrap#1]
Speaking of the problems: LW6.0.1, 32bit, Win7 64 bits.                          

  -- Compilation aborted. Error: Could not load ASDF "2.009" or newer              

  1 (continue) Try loading C:\Users\Art\quicklisp\setup.lisp again.                

  2 Give up loading C:\Users\Art\quicklisp\setup.lisp. --- Error: component  
  "quicklisp" not found                                                            

  1 (continue) Try loading C:\Users\Art\quicklisp\setup.lisp again. ---            

  And lot of warrning dialogs when loading asdf.lisp that goes with  
  quicklisp. Redefining issues.                                                    

  In short, can't get it working :)                                                

  Best,                                                                            

  Art
